Removed db and daoimpl

RLudchenko opened the pull request.

RLudchenko committed:
removed db and daoimpl

dmytro-hryhoruk reviewed (commented):
1)Let's change package naming(in pom.xml) from mate.academy to something more specific to make your project more individual;

2)Let's add README.md with description for your project:
funtionality;
technologies you have used;
guide for set-up: you should provide full instruction how to launch your project on another computer for somebody unfamiliar with your project;

3)DeleteProductsController responsible for deleting only one product. So let's change naming to DeleteProductController. Look around your project for similar cases.

4)While updating order and shopping cart we updating only theirs products. We don't have to update user_id or order/shopping_cart_id.
So we can remove this part.

5)In OrderDao and ShoppingCartDao look in situations when you are updating DB using loop(inserting products mostly).Let's put creating prepare statement for before this loop in order not to create it on each iteration.

6)Check once againt for e.printStackTrace() and replace it with logger.

RLudchenko commented:
@dmytro-hryhoruk 

4) I fixed the update method in order where I removed setting user_id, but I couldn't do it in the shopping cart as it is connected and everything breaks when I try to do this. 

5) I am not sure what you would like me to do? Make prepare statement above the try catch? Did you mean JdbcIpml, not DAO interface? 

6) I use LOGGER everywhere I can, wasn't able to find any printstacktraces()

RLudchenko committed:
removed setup of a user in OrderDaoJdbc update method and changed naming of the main package

dmytro-hryhoruk reviewed (approved):
OrderDaoJdbcImpl line 112 move to 111(before for()) that's what I meant)same in other similar cases.

RLudchenko committed:
changed method getUserOrders and ShoppingCart getByUserId

RLudchenko committed:
fixed mvn errors

RLudchenko committed:
moved prepareStatement to be before for statement

Sofasmile commented on README.md:
```diff
@@ -1,10 +1,31 @@
-"# internet-shop" 
+"# internet-shop"
+#Project Description 
```
```suggestion
# Project Description 
```

Sofasmile commented on README.md:
```diff
@@ -1,10 +1,31 @@
+#Build Pattern 
+This project was built using MVC pattern, it is structured using Dao and Service 
+layers which make it easier to read for other developers.
+**_DAO_** layer contains basic CRUD operations with the persistence layer  
```
```suggestion
- **_DAO_** layer contains basic CRUD operations with the persistence layer  
```

Sofasmile commented on README.md:
```diff
@@ -1,10 +1,31 @@
+This project was built using MVC pattern, it is structured using Dao and Service 
+layers which make it easier to read for other developers.
+**_DAO_** layer contains basic CRUD operations with the persistence layer  
+**_Service_** layer contains business logic for the application 
```
```suggestion
- **_Service_** layer contains business logic for the application 
```

Sofasmile commented on README.md:
```diff
@@ -1,10 +1,31 @@
+layers which make it easier to read for other developers.
+**_DAO_** layer contains basic CRUD operations with the persistence layer  
+**_Service_** layer contains business logic for the application 
+**_Servlets_**, implement client-server communicating logic 
```
```suggestion
- **_Servlets_**, implement client-server communicating logic 
```

RLudchenko committed:
changed readme file

Sofasmile commented on README.md:
```diff
@@ -1 +1,31 @@
-"# internet-shop" 
+"# internet-shop"
```
```suggestion
# internet-shop
```

RLudchenko committed:
changed readme file so it's easier to read

RLudchenko committed:
changed groupId in pom and removed inject field from site
